internal/status/status.go

```diff
@@ -60,6 +61,33 @@ func (s Status) GiveClassName() string {
 	return "Unassigned"
 }
 
+func CodeClassFromName(name string) (int, bool) {
+	switch strings.ToLower(name) {
+	case "1xx":
+		fallthrough
+	case "informal":
+		return Informal, true
```

Comment:
I would prefer to combine each class in its own case, like:

```go
case "1xx", "informal":
  return Informal, true
``` 

wdyt?

Reply:
Great suggestion, @sterchelen! Apologies, my Go has been a bit rusty these days…